src/components/user-list/user-list-styles.ts

```diff
@@ -0,0 +1,40 @@
+import { styled } from 'styled-components';
+import { USER_LIST_BORDER, USER_LIST_BORDER_RADIUS } from '../../utils/constants';
+
+interface UserItemContainerProps {
+  hasBorderBottom: boolean;
+}
+
+const PADDING = '12px';
+const GAP = '8px';
+export const UserListContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  border: ${USER_LIST_BORDER};
+  border-radius: ${USER_LIST_BORDER_RADIUS};
+  .title {
+    background: blueviolet;
+    padding: ${PADDING};
+    border-radius: ${USER_LIST_BORDER_RADIUS} ${USER_LIST_BORDER_RADIUS} 0 0;
+  }
+  h1 {
+    text-align: center;
+    margin: 0;
+    color: white;
+  }
+`;
+
+export const UserItemContainer = styled.div<UserItemContainerProps>`
+  display: flex;
+  flex-direction: column;
+  border-bottom: ${({ hasBorderBottom }) =>
```

Comment:
Se o seu `hasBorderBottom` for false, esse estilo vai chegar no DOM quebrado, algo como:
**<strike>border-bottom: ;</strike>**  
Para que ele não chegue assim quando for false vc pode colocar o estilo dentro da condição, assim ele só altera e vai para o DOM quando for verdadeiro.
```suggestion
 ${({ hasBorderBottom }) => hasBorderBottom &&  border-bottom: 
```